pilot/pkg/networking/core/v1alpha3/cluster_tls.go

```diff
@@ -212,12 +212,21 @@ func (cb *ClusterBuilder) buildUpstreamClusterTLSContext(opts *buildClusterOpts,
 			if !res.IsRootCertificate() || tls.GetInsecureSkipVerify().GetValue() {
 				tlsContext.CommonTlsContext.ValidationContextType = &tlsv3.CommonTlsContext_ValidationContext{}
 			} else {
+				defaultValidationContext := &tlsv3.CertificateValidationContext{MatchSubjectAltNames: util.StringToExactMatch(tls.SubjectAltNames)}
+				if tls.GetCaCrl() != "" {
+					defaultValidationContext.Crl = &core.DataSource{
+						Specifier: &core.DataSource_Filename{
+							Filename: tls.GetCaCrl(),
```

Comment:
how do user put this files to a pod on the fly

Reply:
We build tooling to place certs/crls in the pod and replace it when they change.